diff --git a/app.py b/app.py
index a9018f6..26c364d 100644
--- a/app.py
+++ b/app.py
@@ -107,7 +107,7 @@ def import_history(history, file):
     # The history is returned and will be set to the chatbot component
     return history
 
-with gr.Blocks() as demo:
+with gr.Blocks(delete_cache=(86400, 86400)) as demo:
     gr.Markdown("# Amazon™️ Bedrock™️ Chat™️ (Nils' Version™️) feat. Mistral™️ AI & Anthropic™️ Claude™️")
 
     with gr.Accordion("Startup"):
diff --git a/requirements.txt b/requirements.txt
index 05d9061..bdb8b65 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -1,4 +1,4 @@
-gradio
+gradio>=4.20
 langchain
 boto3>1.34.54
 lxml
\ No newline at end of file
